source/_integrations/unifi.markdown

```diff
@@ -112,6 +112,13 @@ Entities appear for each WLAN. Changing the state of WLAN will trigger a reconfi
 
 ## Sensor
 
+These Sensor are disabled by default, to enable them, click the "Configure" button on the Unifi-integration page and go to page 3/3. Here the different sensors can be enabled.
+
+**Warning:** This will add a new Device to Home Assistant for every connected client in your network. If you network is very large, this will get very messy in Home Assistant.
```

Comment:
```suggestion
```
Moved this information up. I think it is worth mentioning this before showing how to enable the sensors.